rclcpp_examples/test/test_executables_example.py.in

```diff
@@ -30,7 +31,20 @@ def test_executable():
         else:
             exit_handler = ignore_exit_handler
             exit_on_match = False
-        handler = create_handler(name, launch_descriptor, output_file, exit_on_match=exit_on_match)
+
+        filtered_prefixes = [
+            b'pid', b'rc',
+        ]
+
+        if rmw_implementation == 'rmw_connext_cpp' \
+            or rmw_implementation == 'rmw_connext_dynamic_cpp':
+            filtered_prefixes.extend([
+                b'RTI Data Distribution Service',
+                b'Expires on',
+            ])
+            
+        handler = create_handler(name, launch_descriptor, output_file, \
```

Comment:
You don't need `\` within parenthesis to wrap a line.

Usually we try to avoid the aligned indentation because if e.g. you change `handler` to `handler2` you need to update the following lines too. You can just wrap after the open parenthesis:

    handler = create_handler(
        name, launch_descriptor, output_file,
        exit_on_match=exit_on_match, filtered_prefixes=filtered_prefixes)